cppcache/src/PdxType.hpp

```diff
@@ -43,10 +43,11 @@ typedef std::map<std::string, std::shared_ptr<PdxFieldType>> NameVsPdxType;
 class PdxType;
 class PdxTypeRegistry;
 
-class PdxType : public internal::DataSerializableInternal,
-                public std::enable_shared_from_this<PdxType>,
-                private NonCopyable,
-                private NonAssignable {
+class PdxType
```

Comment:
If this is a format only change maybe make the changes necessary to also remove the NonCopyable/Assignable debt.

Reply:
Gonna merge this today as is, but have reliable info that there exists a separate ticket to fix this more generally.